shared/actions/chat.js

```diff
@@ -431,6 +466,10 @@ function * _loadMoreMessages (): SagaGenerator<any, any> {
       paginationNext: pagination.next,
     },
   })
+
+  // Load previews for attachments
+  const attachmentsOnly = messages.reduce((acc: List<Constants.AttachmentMessage>, m) => m && m.type === 'Attachment' ? acc.push(m) : acc, new List())
+  yield attachmentsOnly.map(({conversationIDKey, messageID, filename}: Constants.AttachmentMessage) => put(loadAttachment(conversationIDKey, messageID, true, tmpFile(filename)))).toArray()
```

Comment:
We'll probably need to do something when I make a chat with 1000 images just to annoy you ;)